go.mod

```diff
@@ -11,34 +11,34 @@ require (
 	github.com/ghodss/yaml v1.0.0
 	github.com/go-air/gini v1.0.4
 	github.com/go-bindata/go-bindata/v3 v3.1.3
-	github.com/go-logr/logr v1.2.4
+	github.com/go-logr/logr v1.3.0
 	github.com/golang/mock v1.6.0
-	github.com/google/go-cmp v0.5.9
+	github.com/google/go-cmp v0.6.0
 	github.com/googleapis/gnostic v0.5.5
 	github.com/itchyny/gojq v0.11.0
 	github.com/maxbrunsfeld/counterfeiter/v6 v6.2.2
 	github.com/mikefarah/yq/v3 v3.0.0-20201202084205-8846255d1c37
 	github.com/mitchellh/hashstructure v1.0.0
-	github.com/mitchellh/mapstructure v1.4.1
-	github.com/onsi/ginkgo/v2 v2.9.5
-	github.com/onsi/gomega v1.27.7
+	github.com/mitchellh/mapstructure v1.5.0
+	github.com/onsi/ginkgo/v2 v2.12.0
+	github.com/onsi/gomega v1.27.10
 	github.com/openshift/api v3.9.0+incompatible
 	github.com/openshift/client-go v0.0.0-20220525160904-9e1acff93e4a
 	github.com/operator-framework/api v0.19.0
-	github.com/operator-framework/operator-registry v1.32.0
+	github.com/operator-framework/operator-registry v1.32.1-0.20231119153434-5e23ef594a41
```

Comment:
Any reason we need this pre-release tag?  v1.32.0 has opentelemetry at v1.20 already.

Reply:
/hold to check out the number here

Reply:
opentelemetry was updated in the last operator-registry commit which is not included in v1.32.0

Reply:
Are you sure? The tip commit of operator-registry updates opentelemetry:
https://github.com/operator-framework/operator-registry/commit/5e23ef594a41e6c8ce843d48b22715319c684dff